### DISCUSSION TITLE
violentmonkey扩展添加编辑功能建议

### DISCUSSION THREAD

**Discussion by violenttamper** [violentmonkey/violentmonkey#1276] [Ideas]
希望编写脚本的设置里能加入一个撤销选项，这一定对编写脚本修改很有帮助

**Comment by gera2ld** [comment]
撤销啥？啥选项？

请描述清楚问题，什么场景，遇到了什么问题，目前是什么样子，希望得到什么效果。
标题最好能一句话概括内容，不然每个帖子都要点进来才知道内容，不方便检索和快速查找，对其他人不友好。
如果只是很随意的聊天式的讨论，建议去 Gitter 或者 Reddit。

**Reply by violenttamper** [reply]
首先抱歉没表述清楚，然后我想建议的是：在新建脚本进行脚本编写的时候，能有一个设置选项可以进行"撤销"的操作，类似这个![-3509791bf5413fac.png](https://user-images.githubusercontent.com/75355896/118152275-17359f80-b447-11eb-8dd5-63b57754750b.png)，这对使用violentmonkey扩展编写或修改脚本有些帮助，因为可能violentmonkey扩展是个对用户方便使用，而对编写者可能有些不便，加了也许会变得方便，不加依然简洁。我更希望作者大大能考虑下。像安卓与iOS之间功能的相互"借鉴"，使得体验变得更好。也许因此能有更多新用户，新编写者的使用。
最后感谢作者大大能看到这里。
